pkg/policy/mapstate.go

```diff
@@ -61,6 +63,10 @@ func (k Key) IsEgress() bool {
 	return k.TrafficDirection == trafficdirection.Egress.Uint8()
 }
 
+func (k Key) String() string {
+	return fmt.Sprintf("<Direction=%d, Identity=%d, DestPort=%d, NextHdr=%d>", k.TrafficDirection, k.Identity, k.DestPort, k.Nexthdr)
```

Comment:
Is this change related to the rest of your changes? I was surprised to see it here.

Reply:
I added it for debugging. I'll push it out separately.

Reply:
Still here